src/components/UserInfo.js

```diff
@@ -15,4 +16,8 @@ export default class UserInfo {
     this._titleElement.textContent = title;
     this._descriptionElement.textContent = description;
   }
+
+  setAvatar(avatar) {
```

Comment:
It's great to have a separate method for updating the avatar, since it's controlled by a different form